### DISCUSSION TITLE
Better notes in Auth

### DISCUSSION THREAD

**Discussion by brog-io** [ente-io/ente#3408] [Enhancements]
Currently notes are a bit hidden, you have to edit the entry to view the code which isn't ideal. This could be solved by showing the note above or under the code. But this might be a bit messy

So a button to show the note would be useful. Or maybe even swipe to view the note? Now that everything is done with long press there is space for that

**Comment by ua741** [comment]
Can some please share example how does it look like in other apps that supports note?

**Reply by brog-io** [reply]
It looks like this in 2FAS, but in Ente we already have the email field

![Screenshot_20240924_162002_2FAS Auth](https://github.com/user-attachments/assets/39a96cdf-e5db-4b71-b73c-414a55ebe177)

**Comment by haritvsmurali** [comment]
or even while long pressing edit, share menu was shown. atleast below that would help.
